packages/frontend/src/router.definition.ts

```diff
@@ -105,10 +120,6 @@ const routes: RouteDef[] = [{
 		path: '/theme',
 		name: 'theme',
 		component: page(() => import('@/pages/settings/theme.vue')),
-	}, {
-		path: '/appearance',
-		name: 'appearance',
-		component: page(() => import('@/pages/settings/appearance.vue')),
 	}, {
```

Comment:
同様にですがこれに関しては分散してるのでリダイレクトが正しいと言い切ることは少しむずかしいかもしれない

```suggestion
	}, {
		path: '/appearance',
		redirect: '/settings/preferences',
	}, {
```